app/query_objects/active_query.rb

```diff
@@ -0,0 +1,11 @@
+class ActiveQuery
+  delegate :all, :each, :for_each, to: :active
+
+  def initialize(model)
+    @relation = model.where(active: true)
+  end
+
+  def active
+    @relation
+  end
+end
```

Comment:
`SomeModel.active` is a common query, so have abstracted it out into a query object.